test/e2e/suite/command/sign.go

```diff
@@ -0,0 +1,29 @@
+package command
+
+import (
+	. "github.com/notaryproject/notation/test/e2e/internal/notation"
+	"github.com/notaryproject/notation/test/e2e/internal/utils"
+	. "github.com/onsi/ginkgo/v2"
+)
+
+var _ = Describe("notation sign", func() {
+	It("with JWS signature format", func() {
+		Host(BaseOptions(), func(notation *utils.ExecOpts, artifact *Artifact, vhost *utils.VirtualHost) {
+			notation.Exec("sign", artifact.ReferenceWithTag(), "--signature-format", "jws").
+				MatchKeyWords("Successfully signed")
+
+			OldNotation().Exec("verify", artifact.ReferenceWithTag()).
```

Comment:
I probably missed this step, have we added the certificate into the trust store before running this verification command?

Reply:
If the options is not mentioned, it uses the BaseOptions() which includes the trust store and trust policy for the basic scenario.
```go
func OldNotation(options ...utils.HostOption) *utils.ExecOpts {
	if len(options) == 0 {
		options = BaseOptions()
	}

	vhost, err := createNotationHost(NotationOldBinPath, options...)
	if err != nil {
		panic(err)
	}

	return vhost.Executor
}
```